__e2e__/__snapshots__/config.test.ts.snap

```diff
@@ -0,0 +1,236 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`shows up current config without additional output 1`] = `
+"ctx:  {
```

Comment:
Where this came from? Looks like a broken snapshot that shouldn't end up here, e.g. when using console.log to debug test

Reply:
Forgot console.log yikes !